R/ResampleDesc.R

```diff
@@ -45,8 +46,15 @@
 #'     \dQuote{Subsample} between 0 and 1. Default is 2 / 3.}
 #'   \item{reps [\code{integer(1)}]}{Repeats for \dQuote{RepCV}. Here \code{iters = folds * reps}.
 #'     Default is 10.}
-#'   \item{folds [\code{integer(1)]}}{Folds in the repeated CV for \code{RepCV}.
+#'   \item{folds [\code{integer(1)}]}{Folds in the repeated CV for \code{RepCV}.
 #'     Here \code{iters = folds * reps}. Default is 10.}
+#'   \item{horizon [\code{integer(1)}]}{Number of observations to forecast for \code{GrowthCV}
+#'    and \code{FixedCV}.}
+#'   \item{initial.window [\code{integer(1)}]}{Initial number of observations to start with
+#'    in \code{GrowthCV} and \code{FixedCV}.}
```

Comment:
`initial.window` seems to be a *fraction* of the data size, not the *number* of observations. I assume you changed your mind about this halfway through developing, since `GrowingCV` still has `initial.window = 10` as default.

Also, docu should mention the default.